scripts/install-argocd.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Find the latest ArgoCD version
+VERSION=$(curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
+echo $VERSION
+
+# Platform check
+PLATFORM=`uname -s | awk '{print tolower($0)}'`
+echo $PLATFORM 
+
+# Fetch the appropriate OS binary for ArgoCD
+curl -sSL -o argocd https://github.com/argoproj/argo-cd/releases/download/$VERSION/argocd-$PLATFORM-amd64
```

Comment:
Pr https://github.com/redhat-developer/kam/pull/69 created a bin dir for kam binary. Lets put the argocd binary in same directory to avoid another PATH set for argocd in ```openshiftci-presubmit-all-tests.sh```.

```curl -sSL -o bin/argocd https://github.com/argoproj/argo-cd/releases/download/$VERSION/argocd-$PLATFORM-amd64```